### DISCUSSION TITLE
LiveCharts2 support for native .net6 iOS and Android

### DISCUSSION THREAD

**Discussion by maiiaKey** [Live-Charts/LiveCharts2#621] [General]
Dear LiveCharts2 Team,
We would like to know if it is planned in any timeframe to create a nuget for native .net6 iOS & Android platforms?
We also tried using MAUI controls and then converting them to native iOS control, but this effort didn't succeeded because of MAUI bug with DesignMode.IsDesignModeEnabled: https://github.com/dotnet/maui/issues/7350.

Thank you in advance for your answer.
Maiia

**Comment by zombers** [comment]
I have the same problem, have a solution better yet? @maiiaKey

**Reply by zombers** [reply]
@maiiaKey U just need add " .UseSkiaSharp();" in MauiProgram.cs 
![1663134607590](https://user-images.githubusercontent.com/26043286/190070159-3e9e5d4d-97a1-4676-9a58-c0acbdc6d9d1.png)
